code/API_definitions/device-status.yaml

```diff
@@ -168,7 +354,7 @@ components:
       description: The Mobile country code (MCC) as an geographic region identifier for the country and the dependent areas.
       type: integer
     CountryName:
-      description: The ISO 3166 ALPHA-2 country-codes of mapped to mobile country code(MCC). If there is mapping of one MCC to multiple countries, then we have list of countries. If there is no mapping of MCC to any country, then an empty array [] shall be returned.
+      description: The ISO 3166 ALPHA-2 country-codes of mapped to mobile country code(MCC). If there is mapping of one MCC to multiple countries, then we have list of countries. If there is no mapping of MCC to any country, then ["xx"] will be used.
```

Comment:
@bigludo7 I forgot to mention this in the meeting. Here it's overwriting the last change that was added to the main branch. Could you please align it?